scripts/utils.sh

```diff
@@ -8,6 +8,7 @@ function setup_folders() {
     mkdir -p ~/.firewalla/run
     mkdir -p ~/.forever
     mkdir -p ~/logs
+    sudo chown -R pi ~/logs
```

Comment:
minor, better use `sudo chown -R pi ~/logs/`

the logs folder is a symlink, "-R" does follow symlink, but add / is for better reading.